src/parse.js

```diff
@@ -477,6 +477,74 @@ Parser.prototype.parseCoverage = function() {
     throw new Error('0x' + startOffset.toString(16) + ': Coverage format must be 1 or 2.');
 };
 
+/**
+ * Parse a BaseArray Table in GPOS table
+ * https://learn.microsoft.com/en-us/typography/opentype/otspec191alpha/gpos#lookup-type-4-mark-to-base-attachment-positioning-subtable
+ *
+ * @param {Number} marksClassCount
+ * @returns {Array}
+ */
+Parser.prototype.parseBaseArray = function(marksClassCount) {
+    const count = this.parseUShort();
+    return this.parseList(count, Parser.list(
+        marksClassCount,
+        Parser.pointer(Parser.anchor)
+    ));
+};
+
+/**
+ * Parse a MarkArray Table in GPOS table
+ * https://learn.microsoft.com/en-us/typography/opentype/otspec191alpha/gpos_delta#mark-array-table
+ *
+ * @returns {Array}
+ */
+Parser.prototype.parseMarkArray = function() {
+    const count = this.parseUShort();
+    return this.parseRecordList(count, {
+        class: Parser.uShort,
+        attachmentPoint: Parser.pointer(Parser.anchor)
+    });
+};
+
+/**
+ * Parse a an anchor definition Table in GPOS table
+ * https://learn.microsoft.com/en-us/typography/opentype/otspec191alpha/gpos_delta#anchor-tables
+ *
+ * @returns {Object} Anchor object representing format type
+ */
+Parser.prototype.parseAnchorPoint = function() {
+    const startOffset = this.offset + this.relativeOffset;
+    const format = this.parseUShort();
+    switch (format) {
+        case 1:
+            return {
+                format,
+                xCoordinate: this.parseShort(),
+                yCoordinate: this.parseShort()
+            };
+        case 2:
+            return {
+                format,
+                xCoordinate: this.parseShort(),
+                yCoordinate: this.parseShort(),
+                anchorPoint: this.parseUShort()
+            };
+
+        // TODO: Add a support Device offsets
```

Comment:
As with the other TODO, has this been accidentally forgotten, or why is it not implemented with this PR? (not supposed to sound like an accusation/demand, just would like to know what's the reason)

Reply:
@Connum it was intended, because actually this was not required to be supported within the iteration. I covered the planned scope which was not including this bit. If you are concerned, I think I might revisit this

Reply:
Assuming this comment https://github.com/opentypejs/opentype.js/pull/557#issuecomment-1453904466 was referring to this?:) Please resolve thread if we are on the same page. Thanks.